tests/test_api_endpoints.py

```diff
@@ -0,0 +1,219 @@
+import os
+import json
+import base64
+import psycopg2
+import requests
+from contextlib import closing
+
+BASE_URL = "http://localhost:5000"
+USERNAME = 'simulator'
+PWD = 'super_safe!'
+CREDENTIALS = ':'.join([USERNAME, PWD]).encode('ascii')
+ENCODED_CREDENTIALS = base64.b64encode(CREDENTIALS).decode()
+HEADERS = {'Connection': 'close',
+           'Content-Type': 'application/json',
+           f'Authorization': f'Basic {ENCODED_CREDENTIALS}'}
+
+# Get the database URL from the environment variable
+DATABASE_URL = "postgresql://user:pass@localhost:5432/waect" #os.getenv("DATABASE_URL")
```

Comment:
I had problems running the `python-flask` test here. I had to use the ip from `ipconfig` again. Do you have a clever solution for this? Also let's load the URL from the `.env` file here?

Reply:
Thats sounds a bit odd. Lets check it tomorrow. Maybe something is wrong with the ports your mac is using. Yes, we gonna load the URL from the .env file.